resources/benchmark-runner.mjs

```diff
@@ -574,9 +574,9 @@ export class BenchmarkRunner {
             // Prepare all iteration metrics so they are listed at the end of
             // of the _metrics object, before "Total" and "Score".
             for (let i = 0; i < this._iterationCount; i++)
-                iterationTotalMetric(i);
-            getMetric("Geomean");
-            getMetric("Score", "score");
+                iterationTotalMetric(i).description = `Test totals for iteration ${i}`;
+            getMetric("Geomean", "ms").description = "Geomean of test totals";
+            getMetric("Score", "score").description = "Scaled inverse of the Geomean";
```

Comment:
You're not doing anything with these `description` properties, aren't you?

Reply:
Correct, this is just for them to show up the serialized JSON as sort of an explainer.

Reply:
oh I see, makes sense

Reply:
I might add a help-text  / hover alt text in the future with the description.